ibis/expr/types/relations.py

```diff
@@ -2957,6 +2958,415 @@ def pivot_longer(
 
         return self.select(~pivot_sel, **new_cols)
 
+    @util.experimental
+    def pivot_wider(
+        self,
+        *,
+        id_cols: s.Selector | None = None,
+        names_from: str | Iterable[str | s.Selector] = "name",
+        names: Iterable[str] | None = None,
+        names_prefix: str = "",
+        names_sep: str = "_",
+        names_sort: bool = False,
```

Comment:
Do we need the `names_` prefixes?  I'm leaning towards not having them.

Reply:
I actually prefer them so that I know the thing the argument is applied to. IMO it's not obvious that `prefix`, `sep`, and `sort` would apply to column names and not the resulting values.